custom/icds_reports/dashboard_utils.py

```diff
@@ -46,9 +46,11 @@ def get_dashboard_template_context(domain, couch_user):
     if couch_user.is_web_user():
         context['is_web_user'] = True
 
+    beta = context['have_access_to_features']
```

Comment:
nit: probably a bit clearer to make this a local variable before it's used the first time, rather than assume it from the context.